pkg/test/framework/components/echo/common/deployment/echos.go

```diff
@@ -63,7 +63,7 @@ type Config struct {
 	//
 	// Custom echo instances will be accessible from the `All` field in the namespace(s) under which they
 	// were created.
-	Custom []echo.Config
+	Custom echo.CustomGetter
```

Comment:
So everytime when we call this from `deployment.SetupSingleNamespace` and `deployment.SetupTwoNamespaces`, we would require to call it this way only otherwise it won't accept it. So IMO it's good to have it like this. We are already doing it for namespace in the same file. Let me know what do you think ?

Reply:
I'm not against, TBH ... just curious. I guess it allows you to just pass a common method.